activerecord/lib/active_record/test_fixtures.rb

```diff
@@ -67,6 +67,7 @@ def setup_fixture_accessors(fixture_set_names = nil)
         end
       end
 
+      # Makes the given test methods use real transactions instead of transactional tests.
```

Comment:
I feel like we could be more clear about the purpose of this method.  What about the following?

```suggestion
      # Prevents automatically wrapping each specified test in a transaction,
      # to allow application logic transactions to be tested in a top-level
      # (non-nested) context.
```

Reply:
Wow thanks! I was not so sure how concrete it could read to be honest. This looks much better to me and I'd like to commit your suggestion 😄